tests/test_evaluation.py

```diff
@@ -0,0 +1,75 @@
+import polars as pl
+import numpy as np
+import iup
+from iup import eval
+import pytest
+from datetime import date
+
+
+@pytest.fixture
```

Comment:
Because we now only use the `data()` and `pred()` fixtures for one test, you could just move the definition of those data frames into that one test

but not super high priority at all